CHANGELOG.md

```diff
@@ -12,9 +12,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Documented how to use layouts (see [issue 63](https://github.com/KNowledgeOnWebScale/walder/issues/63))
 - Supporting comunica lenient querying (see [issue 68](https://github.com/KNowledgeOnWebScale/walder/issues/68))
 - FrontMatter metadata available in layout templates (see [issue 72](https://github.com/KNowledgeOnWebScale/walder/issues/72))
+- Logging to HtmlConverter
+- Default support for Nunjucks templates
 
 ### Changed
 - Update dependencies
+- Methods of HtmlConverter are non-static anymore
```

Comment:
Solves 1 of 2 converters mentioned in #58. Only rdf-converter to go. Suggestion to do this one too. Perhaps also with some logging...

Reply:
I would stick to only this one because of [scope creep](https://en.wikipedia.org/wiki/Scope_creep).